Extend sysID with parsing tools and some scripts to quantify IMU quality

helenol opened the pull request:
Results of these in:
https://github.com/ethz-asl/mav_tools/wiki/IMU-Noise-Analysis

fmina commented on read_bags/readAttitudeTarget.m:
```diff
@@ -0,0 +1,17 @@
+function tf = readAttitudeTarget(bag, topic)
```
maybe change `tf` to something like `targetAttitude`.

helenol commented:
@fmina thanks! Now updated to the NEWEST of Jay params.
FYI I moved read**Commnad**PitchRollYawRateThrust to read**Command**PitchRollYawRateThrust so if your stuff breaks it's the rename. 🙈
